backend/howtheyvote/store/mappings.py

```diff
@@ -62,6 +62,9 @@ def map_vote(record: CompositeRecord) -> Vote:
     eurovoc_concepts = {EurovocConcept[id_] for id_ in record.chain("eurovoc_concepts")}
     responsible_committee = Committee.get(record.first("responsible_committee"))
 
+    press_release = record.first("press_release")
+    is_featured = record.first("is_featured") or press_release is not None
```

Comment:
Not sure we still need the `is_featured` column in the future.

Reply:
Decided to do this in a separate PR so not to make the diff even larger than it already is